Source/engine/events.cpp

```diff
@@ -120,25 +121,6 @@ bool FetchMessage_Real(SDL_Event *event, uint16_t *modState)
 		*event = e;
 		break;
 #ifndef USE_SDL1
-	case SDL_MOUSEWHEEL:
-#ifdef _DEBUG
-		if (IsConsoleOpen()) {
-			*event = e;
-			break;
-		}
-#endif
-		// This is a hack, mousewheel events should be handled directly by their consumers instead.
-		event->type = SDL_KEYDOWN;
-		if (e.wheel.y > 0) {
-			event->key.keysym.sym = (SDL_GetModState() & KMOD_CTRL) != 0 ? SDLK_KP_PLUS : SDLK_UP;

```

Comment:
I'm super happy to see this gone, thank you! :tada:
Did we lose the Control + Wheel functionality to zoom the map with this change though?

Reply:
Yes this logic is also removed. You can still zoom with the +/- keys.
But if we want this back as special case, we can implement them the same way as scrolling in shops etc. is done. 🙂